### DISCUSSION TITLE
Information about who is the origin of each reaction

### DISCUSSION THREAD

**Discussion by arturo-ges** [Tyrrrz/DiscordChatExporter#1044] [Q&A] [answered]
Dear all,

Is there any way to get into the exported files information about who is the origin of each reaction?

In the Discord application you can know who has given each reaction to a message by stopping the pointer on the desired reaction symbol. We can not find this information in the exported files. 

It would be very useful for us.

**Comment by Tyrrrz** [comment, marked as answer]
#133

**Reply by arturo-ges** [reply]
Thanks a lot for the pointer to the previous thread. I missed it when I was doing the search.
I see the problem :-(
